src/rgw/rgw_d3n_cacherequest.h

```diff
@@ -137,12 +139,18 @@ struct D3nL1CacheRequest {
     }
   };
 
+  static std::string generate_oid_digest(const std::string& oid) {
+    XXH128_hash_t hash = XXH3_128bits(oid.c_str(), oid.size());
+    std::string digest = fmt::format("{:016x}{:016x}", hash.high64, hash.low64);
```

Comment:
@cbodley force-pushed for refactoring
snprintf(digest, sizeof(digest), "%016lx%016lx", hash.high64, hash.low64);
to
fmt::format("{:016x}{:016x}", hash.high64, hash.low64);